topics/07_web_ui_testing/ui_text_box_page.py

```diff
@@ -0,0 +1,45 @@
+from selenium.webdriver.common.by import By
+from selenium.webdriver.common.keys import Keys
+
+
+class TextBoxPage:
+    def __init__(self, driver):
+        self.driver = driver
+        self.url = "https://demoqa.com/text-box"
+
+        """Page locators"""
```

Comment:
Triple quotes are typically used for docstrings and multiline comments, whereas `#` is customary for single-line comments.